sqlglot/__init__.py

```diff
@@ -22,6 +22,7 @@
     or_,
     select,
     subquery,
+    DataType,
```

Comment:
hm not sure about this one, we don’t expose any others. thoughts?

Reply:
I don't feel strongly either way. To me it seemed like DataType would be a common expressions users would want and therefore was as common/important as these other expressions. It is not a big deal to switch though.

Reply:
yea let’s get rid of it